packages/service-clients/end-to-end-tests/azure-client/src/test/AzureClientFactory.ts

```diff
@@ -70,3 +71,60 @@ export function createAzureClient(
 		configProvider,
 	});
 }
+
+export async function createContainerFromPayload(
```

Comment:
Please add a tsdoc comment here about the usage and why it's needed for ephemeral testing, probably mention the payload file (ephemeralSummaryTrees.ts) as a reference, and also discourage unsanctioned use outside of ephemeral container testing.